cloudformation/media-atom-maker.json

```diff
@@ -523,6 +523,27 @@
                 "Statistic": "Average",
                 "AlarmActions": [ { "Ref": "AlertTopic" } ]
             }
+        },
+        "MediaAtomMakerUnhealthy": {
+          "Type": "AWS::CloudWatch::Alarm",
+          "Properties": {
+            "ActionsEnabled": { "Ref": "AlertActive" },
+            "AlarmDescription": "Unexpectedly unhealthy instances",
+            "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+            "Threshold": "1",
+            "Namespace": "AWS/ELB",
+            "MetricName": "UnHealthyHostCount",
+            "Dimensions": [
+              {
+                "Name": "LoadBalancerName",
+                "Value": { "Ref": "MediaAtomMakerLoadBalancer" }
+              }
+            ],
+            "Period": "300",
+            "EvaluationPeriods": "1",
```

Comment:
As with all the alarms, we should knock it down if it gets spammy. I think it's best to be high whilst in the trial